docs/how-we-work.md

```diff
@@ -72,6 +72,18 @@ Major, cross-cutting refactoring efforts fit within this category. Our goals wit
 
 To introduce brand-new functionality into the package, follow this guide.
 
+##### On using RFCs
+
+We use a lightweight RFC (request for comments) process to ensure that folks have an opportunity to weigh in on design, alternatives, drawbacks, questions, and concerns. It provides a quick and easily scannable summary of what was discussed and decided.
```

Comment:
One more thing that caught my eye:

```suggestion
We use an RFC (request for comments) process to ensure that folks have an opportunity to weigh in on design, alternatives, drawbacks, questions, and concerns. It provides a quick and easily scannable summary of what was discussed and decided.
```

Calling our own RFC process "lightweight" feels like calling our software "simple" in the README - lightweight-ness is in the eye of the beholder :wink: We _want_ it to be lightweight, but maybe we shouldn't just say that it is... ?

Reply:
Heh yeah I hear you. I was trying to find a way to make RFC sound more approachable. I think the term "RFC" itself sounds intimidating and it could be reassuring to have some other descriptive word to keep people's imagination from making it bigger and scarier than it really is. But yeah, everything is relative and we can't objectively call this lightweight. I'm fine taking it out. Perhaps you have another suggestion for how to convey to folks that the scope isn't this [super formal daunting thing](http://encyclopedia2.thefreedictionary.com/Request+For+Comment) that might deter them?